static/app/views/organizationDetails/index.tsx

```diff
@@ -120,29 +120,30 @@ class DeletionPending extends Component<PendingProps, PendingState> {
   }
 }
 
-class OrganizationDetailsBody extends Component {
-  static contextTypes = {
-    organization: SentryTypes.Organization,
-  };
+type OrganizationDetailsProps = {
+  organization?: Organization;
+  children?: React.ReactNode;
+};
 
-  render() {
-    const {organization} = this.context;
-
-    if (organization && organization.status) {
-      if (organization.status.id === 'pending_deletion') {
-        return <DeletionPending organization={organization} />;
-      } else if (organization.status.id === 'deletion_in_progress') {
-        return <DeletionInProgress organization={organization} />;
-      }
+const OrganizationDetailsBody = withOrganization(function OrganizationDetailsBody({
+  children,
+  organization,
+}: OrganizationDetailsProps) {
+  if (organization?.status) {
```

Comment:
Actualy I guess you don't even really need this since you can write the following if's with `organization?.status?.id` lol

Reply:
I can trim this down in a follow up pull request as I have one ready to replace the nested layouts.